app/assets/javascripts/app/manage_orders.coffee

```diff
@@ -86,6 +86,12 @@ class OrderDetailManagement
       total = row.find('.cost input').val() - row.find('.subsidy input').val()
       row.find('.total input').val(total.toFixed(2))
       self.notify_of_update $(row).find('input[name*=total]')
+      enablePriceChangeReasonField()
+
+  enablePriceChangeReasonField = ->
```

Comment:
I'm not quite set on this, but I feel like we shouldn't even show the field until you adjust the price. And then it should go away if the price is recalculated. My gut feeling is the enabled status should be based on field value `!=` calculated (i.e. expected) value.